### ISSUE TITLE
Unrecognized escape \R

### ISSUE THREAD

**Issue by EricDeveaud** [harry-thorpe/piggy#12]
hello,

I have various warning while running piggy

```
Unrecognized escape \R passed through at /local/gensoft2/exe/piggy/1.0/libexec/piggy_core_alignment_creator.pl line 34.
```

don't have time right now to check it, 
maybe https://stackoverflow.com/questions/39090265/unrecognized-escape-r-passed-through-at-test-pl-line-7 can help

regards

Eric

**Comment by harry-thorpe** [comment]
Hi Eric,

This is a catchall newline character (\n \r \r\n) implemented in perl 5.10.0. I have changed it to search for these separately, so should work now.

Thanks,

Harry

**Comment by EricDeveaud** [comment]
thanks for all the fixes.

maybe this worth a new release ? 

Eric
